### ISSUE TITLE
What to do to keep prediction the same distribution with ground truth ? Specifically, BCE Loss and MSE Loss cannot make the prediction keep the same distribution with ground truth?

### ISSUE THREAD

**Issue by Leeinsn** [fuxiAIlab/perCLTV#4]

**Comment by Wwwwei** [comment]
Thanks for your attention. You might be interested in **independent and identically distributed (IID)**.

For a machine learning or deep learning problem, this assumption usually needs to be satisfied. That is, it is assumed that the training data and the test data meet the same distribution, which is a basic guarantee that the model obtained through the training data can obtain good results in the test set.

BCE Loss and MSE Loss here are only used to help models optimize themselves from supervised training data.

**Comment by Leeinsn** [comment]
Thanks for your timely response. 

Actually, what I care about is as long as good MAPE / MAE the trained model get, but model cannot fit the trend of the data. For example, in our experiment, a LTV model may get excellent MAPE / MAE in regression part. Nevertheless, relative relationships between labels are not well presented. 
Thanks again for your reply.

If you have any new insights to be shared, I would appreciate it.

~^v^~~~
